src/cljc/rems/common/util.cljc

```diff
@@ -7,6 +7,8 @@
 ;; regex from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation
 (def +email-regex+ #"[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*")
 
+(def +phone-number-regex+ #"^\+\d{1,3}\s\d{2,3}\s\d{2,3}\s\d{4}|^\+\d{1,3}\s\d{1,14}(\s\d{1,13})?$|^\+[1-9]\d{5,14}$\s?$")
```

Comment:
Hmm, why not just `#"^\+[ 0-9]+`: a plus followed by spaces and numbers. We don't need to restrict how people enter spaces in their phone numbers, right?

Could also consider normalizing the phone number by removing spaces when it gets stored in the db (I think Jaakko requsted this), but that can be done in a follow-up PR.

Reply:
i agree with Joel, we should not care how people type in the number, but sanitise it when storing it in the db. this actually makes my previous comment about fixing the max length invalid, since we should allow people to type even "+ 1 2 3 4 5 6 7 8    9 10 11".

Reply:
I could do that here!